src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Properties/InterceptedProjectProperties/ApplicationPropertyPage/CompoundTargetFrameworkValueProvider.cs

```diff
@@ -61,7 +61,7 @@ public override Task<bool> IsValueDefinedInContextAsync(string propertyName, IPr
             return IsValueDefinedInContextMSBuildPropertiesAsync(defaultProperties, s_msBuildPropertyNames);
         }
 
-        public override async Task<string?> OnSetPropertyValueAsync(string propertyName, string unevaluatedPropertyValue, IProjectProperties defaultProperties, IReadOnlyDictionary<string, string>? dimensionalConditions = null)
+        public override async Task<string?> OnSetPropertyValueAsync(string propertyName, string unevaluatedPropertyValue, IProjectProperties projectProperties, IReadOnlyDictionary<string, string>? dimensionalConditions = null)
```

Comment:
Does this rename match the parameter name of the base method?

Reply:
oh it doesn't, let me revert this, thanks for noticing!